source/extensions/filters/http/jwt_authn/authenticator.cc

```diff
@@ -249,8 +209,9 @@ void AuthenticatorImpl::doneWithStatus(const Status& status) {
 
 } // namespace
 
-AuthenticatorPtr Authenticator::create(FilterConfigSharedPtr config) {
-  return std::make_unique<AuthenticatorImpl>(config);
+AuthenticatorPtr Authenticator::create(FilterConfigSharedPtr config,
+                                       Common::JwksFetcher::JwksFetcherPtr& fetcher) {
+  return std::make_unique<AuthenticatorImpl>(config, fetcher);
```

Comment:
Original comment from @qiwzhang on #4225

"should we call std::move for fetcher"